### PR TITLE
WIP: update the config-management-operator

### PR DESCRIPTION
**Pull request by Liujingfang1**

### CODE REVIEW

**Comment by Liujingfang1** [comment]
cc @janetkuo @yuwenma @nan-yu @haiyanmeng

**Review by janetkuo — APPROVED** [review]

**Review by nan-yu — APPROVED** [review]
/lgtm

File: 3_acm/install_acm.sh [hunk 1]
```diff
@@ -52,6 +52,7 @@ for CONTEXT in ${CLUSTERS}; do
     cat config-management.yaml.tpl | envsubst > ${CONFIGMAGFILE}
     kubectl apply -f ${CONFIGMAGFILE}
     rm ${CONFIGMAGFILE}
+    sleep 60
```
Comment (janetkuo) [inline 3_acm/install_acm.sh:55]:
```suggestion
    echo "Sleep for 1 minute to allow RootSync CRD to be created so the RootSync resource can be applied"
    sleep 1m
```

Comment (Liujingfang1) [reply 3_acm/install_acm.sh]: Done

**Review by janetkuo — APPROVED** [review]

**Pull request merged**